packages/desktop-client/src/components/transactions/MobileTransaction.js

```diff
@@ -373,7 +373,7 @@ class TransactionEditInner extends PureComponent {
             }}
           >
             <Link
-              to={`/accounts/${account.id}`}
+              to={account ? `/accounts/${account.id}` : '/accounts'}
```

Comment:
This is the `back` button in transaction entry page.

If there is no account - back should take you to "all accounts" page.